hack/verify-boilerplate.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 PingCAP, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+ROOT=$(unset CDPATH && cd $(dirname "${BASH_SOURCE[0]}")/.. && pwd)
+cd $ROOT
+
+boiler="${ROOT}/hack/boilerplate/boilerplate.py"
+
+#
+# TODO update license information for following files
+#
+# - ./deploy/*
+# - */*.sh
+# - */Makefile
+# - */Dockerfile
+# - */*.register.go # files geneated by apiregister-gen
+#
+files=($(find . -type f -not \( \
+        -path './hack/boilerplate/*' \
+        -o -path './_tools/*' \
+		-o -path './.git/*' \
+		-o -path './vendor/*' \
+		-o -path './pkg/client/*' \
+        -o -path './*/.terraform/*' \
+        -o -path './deploy/*' \
+        -o -path '*/*.sh' \
+        -o -path '*/Makefile' \
+        -o -path '*/Dockerfile' \
+        -o -path '*/*.register.go' \
+    \)
+))
```

Comment:
can we consider gitignore? it is okay for CI but broken in my local development env

Reply:
They use different syntaxes. Which pattern you want to ignore, I can add it.

Reply:
`./.idea/*`

Reply:
fixed in f7306b8c67a5716b08eb7eff779d08ba671b2db6, all hidden files/directories are ignored